spec/dependent_styles_spec.rb

```diff
@@ -1,82 +1,79 @@
-Dependents.each_pair do |id, (filename, path, style, reason)|
+Dependents.each_pair do |basename, (filename, path, style, reason)|
 
-  describe "dependent style #{id}" do
+  describe "#{basename} (dependent):" do
```

Comment:
The colon `:` is most of the times not needed. E.g. 
```4or (dependent) must validate against the CSL 1.0.1 schema```
I think I would delete it again, at least in the default string.